packages/backend/src/controllers/organizationController.ts

```diff
@@ -174,6 +175,50 @@ export class OrganizationController extends Controller {
         };
     }
 
+    /**
+     * Get the member profile for a user in the current user's organization by email
+     * @param req express request
+     * @param primary email of the user
+     */
+    @Middlewares([allowApiKeyAuthentication, isAuthenticated])
+    @Get('/users/emails/{email}')
+    @OperationId('GetOrganizationMemberByEmail')
+    async getOrganizationMemberByEmail(
+        @Request() req: express.Request,
+        @Path() email: string,
+    ): Promise<ApiOrganizationMemberProfile> {
```

Comment:
I wondered if we should make it a GET method, since the path parameter of email appears in access logs and so on. Email addresses can be sensitive. It doesn't look great to make it a POST method. But, that enables us to hide email addresses in access logs and so on. What do you think?

Reply:
Good question. A formal API would usually require this to be tokenized. But that's kind of what we are doing with the UUID versions of this. Do we need the the human readable version?

If so, this probably should be a POST. @owlas I don't know what we like to do here? I've used POST for this reason in the past, but we should be consistent.